eth/vm/forks/petersburg/headers.py

```diff
@@ -0,0 +1,13 @@
+from eth.vm.forks.byzantium.headers import (
+    configure_header,
+    create_header_from_parent,
+    compute_difficulty,
+)
+
+
+compute_petersburg_difficulty = compute_difficulty(5000000)
```

Comment:
Maybe `compute_difficulty(bomb_delay=5000000)` as a way to document what this magic number is.

Reply:
For some reason, this fails the tests with:

>TypeError: compute_difficulty() got multiple values for argument 'bomb_delay' ([CI job](https://circleci.com/gh/ethereum/py-evm/126329?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link))

Leaving it as is for now.